repo/splitter/splitter.go

```diff
@@ -7,6 +7,11 @@ import (
 
 const (
 	splitterSlidingWindowSize = 64
+
+	splitterSize1MB = 1 << 20
+	splitterSize2MB = 2 << 20
+	splitterSize4MB = 4 << 20
+	splitterSize8MB = 8 << 20
```

Comment:
nit: given that this is in the splitter package and not exported, they could be simply named `sizeXMB`

Reply:
I already pushed that commit independently as I was testing linter behavior.